azure-pipelines.yml

```diff
@@ -203,6 +203,7 @@ stages:
             -restore
             -build
             -pack
+            -sign
```

Comment:
That certainly seems correct to me. The one thing is we should probably only sign for the release build right?